android/src/main/java/com/reactnativedidomi/DidomiModule.kt

```diff
@@ -641,7 +642,7 @@ class DidomiModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
                     organizationUserIdAuthDigest
             )
         } else {
-            Didomi.getInstance().setUser(organizationUserId)
+            Didomi.getInstance().setUser(organizationUserId, currentActivity as? FragmentActivity)
```

Comment:
As the activity parameter is optional, we should provide a way to omit it. For Unity, I left the original method and added corresponding new methods `setUserAndSetupUI(....)`

Reply:
will do the changes and add some tests